net-libs/libisds/libisds-0.11.1.ebuild

```diff
@@ -56,5 +51,5 @@ src_configure() {
 src_install() {
 	default
 
-	find "${ED}/" \( -name "*.a" -o -name "*.la" \) -delete || die
```

Comment:
haha, so the package before gave you the option to build static-libs, and then proceeded to delete them unconditionally :joy:

Reply:
Yeah, pretty wild :smile: